config.m4

```diff
@@ -194,6 +200,31 @@ if test "$PHP_REDIS" != "no"; then
     fi
   fi
 
+  if test "$PHP_REDIS_LZ4" != "no"; then
+      AC_DEFINE(HAVE_REDIS_LZ4, 1, [ ])
+      AC_MSG_CHECKING(for liblz4 files in default path)
+      for i in $PHP_LIBLZ4 /usr/local /usr; do
+        if test -r $i/include/lz4.h; then
+          AC_MSG_RESULT(found in $i)
+          LIBLZ4_DIR=$i
+          break
+        fi
+      done
+      if test -z "$LIBLZ4_DIR"; then
+        AC_MSG_RESULT([not found])
+        AC_MSG_ERROR([Please reinstall the liblz4 distribution])
+      fi
+      PHP_CHECK_LIBRARY(lz4, LZ4_compress,
+      [
+        PHP_ADD_LIBRARY_WITH_PATH(zstd, $LIBLZ4_DIR/$PHP_LIBDIR, REDIS_SHARED_LIBADD)
```

Comment:
Should this mention `zstd` here?

Reply:
No but this was fixed in 19269cb1db073376364d2655fc828daeb8f1006e